doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _app_dfu:
+.. _ug_fw_update:
+
+Device Firmware Updates
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Device Firmware Update (DFU) is the procedure of upgrading the application firmware version on a device.
+It is composed of two steps.
+First a new firmware image is transferred to the chip, and then the :ref:`bootloader <app_bootloaders>` tests and boots the new firmware.
+
+You can transfer the updated images to the device in two ways:
+
+* Wired, where updates are sent through a wired connection, like UART, or delivered by connecting a flash device.
+* Over-the-air (OTA), where updates are sent through a wireless connection, like Bluetooth® Low Energy.
+
+The testing and booting process depends on the choice of bootloader.
+Generally, bootloaders support two types of updates:
+
+* Direct updates, with an in-place substitution of the image.
+  In this case, it is the bootloader that must transfer the update image.
+* Background updates, where the updated image is obtained and stored by the application, but the update is completed by the bootloader after the device reboots.
+
+Based on these criteria, the |NCS| offers support for the following DFU alternatives:
+
+* For sending updates to the receiving device:
+
+  .. list-table:: DFU methods for sending updates
+    :widths: auto
+    :header-rows: 1
+
+    * - Method
+      - Description
+      - Supported transfer
+    * - :ref:`nRF Util <toolchain_management_tools>`
+      - Multi-purpose command line tool that can be used to send SMP-formatted updates to MCUmgr, among other things.
+      - Wired (SMP over UART or USB)
+    * - Cloud interfaces
+      - Each cloud comes with its specific interface to send updates to devices from the cloud. See documentation for :ref:`lib_nrf_cloud`, :ref:`lib_aws_fota` or :ref:`lib_azure_fota`.
+      - OTA (LTE, Wi-Fi)
+    * - :ref:`zephyr:mcumgr_cli`
+      - Command line tool specifically for sending updates to the MCUmgr library.
+      - Wired (SMP over UART or USB) and OTA (SMP over Bluetooth LE)
+    * - `nRF Connect Device Manager`_
+      - Mobile application for sending SMP updates over Bluetooth LE; can also be used for other SMP functionality.
+      - OTA (SMP over Bluetooth LE)
+    * - `nRF Connect for Mobile`_
+      - General purpose mobile application that can be used for sending SMP updates over Bluetooth LE, among other things.
+      - OTA (SMP over Bluetooth LE)
+    * - :ref:`MCUmgr SMP client <zephyr:mcumgr_smp_transport_specification>`
+      - SMP Client running on a microcontroller. :ref:`bluetooth_central_dfu_smp` is a good starting point for implementing DFU.
+      - Wired (SMP over UART or USB) and OTA (SMP over Bluetooth LE)
```

Comment:
Isn't this one only for OTA over Bluetooth LE ?

Reply:
Edited as discussed in chat.